PrebidMobile/drprebid/src/main/assets/about.html

```diff
@@ -0,0 +1,46 @@
+<doctype! html>
+
+<html lang="en-US">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Dr. Prebid</title>
+
+  <style>
+    h1   {color: rgb(59, 136, 195); font-size:20x; font-weight: bold; font-family: -apple-system, BlinkMacSystemFont, sans-serif}
```

Comment:
for all the in-app html help files, the font-family should be something for android. I haven't tested yet so I don't know what it looks like with the apple font-family defined.

Reply:
I used the same html files from the Dr. Prebid iOS app. So that's probably why they have the Apple CSS. But I tested them and it looked fine.